Adyen/Model/MarketPay/FieldType.cs

```diff
@@ -48,517 +48,920 @@ public enum FieldNameEnum
             /// </summary>
             [EnumMember(Value = "accountCode")]
```

Comment:
why do all enumMember values in this file start with lowercase? in other files they start with uppercase